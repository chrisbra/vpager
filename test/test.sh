#!/bin/sh

#set -x

# Setup some color variables
red=$(tput setaf 1)
green=$(tput setaf 2)
reset=$(tput sgr0)

for i in */; do
  (
    cd "$i" &&
    test -f "cmd" &&
    ./cmd > test.out &&
    diff -qu test.expected  test.out >/dev/null ;
    if [ $? -ne 0 ]; then
      printf "Test %s:\t\t[${red}Failed${reset}]\n" ${i%%/}
      diff -u test.expected test.out
    else
      printf "Test %s:\t\t[${green}OK${reset}]\n" ${i%%/}
    fi
    )
done
