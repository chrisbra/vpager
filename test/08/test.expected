]51;["call", "Tapi_Vpager_setup", ["vpager_setup", "%d _","setl ft=diff"]]]51;["call", "Tapi_Vpager",  ["vpager", "1"]]1
]51;["call", "Tapi_Vpager",  ["vpager", "2"]]2
]51;["call", "Tapi_Vpager",  ["vpager", "3"]]3
]51;["call", "Tapi_Vpager",  ["vpager", "4"]]4
]51;["call", "Tapi_Vpager",  ["vpager", "5"]]5
]51;["call", "Tapi_Vpager",  ["vpager", "6"]]6
]51;["call", "Tapi_Vpager",  ["vpager", "7"]]7
]51;["call", "Tapi_Vpager",  ["vpager", "8"]]8
]51;["call", "Tapi_Vpager",  ["vpager", "9"]]9
]51;["call", "Tapi_Vpager",  ["vpager", "10"]]10
]51;["call", "Tapi_Vpager",  ["vpager", "11"]]11
]51;["call", "Tapi_Vpager",  ["vpager", "12"]]12
]51;["call", "Tapi_Vpager",  ["vpager", "13"]]13
]51;["call", "Tapi_Vpager",  ["vpager", "14"]]14
]51;["call", "Tapi_Vpager",  ["vpager", "15"]]15
]51;["call", "Tapi_Vpager",  ["vpager", "16"]]16
]51;["call", "Tapi_Vpager",  ["vpager", "17"]]17
]51;["call", "Tapi_Vpager",  ["vpager", "18"]]18
]51;["call", "Tapi_Vpager",  ["vpager", "19"]]19
]51;["call", "Tapi_Vpager",  ["vpager", "20"]]20
]51;["call", "Tapi_Vpager",  ["vpager", "21"]]21
]51;["call", "Tapi_Vpager",  ["vpager", "22"]]22
]51;["call", "Tapi_Vpager",  ["vpager", "23"]]23
]51;["call", "Tapi_Vpager",  ["vpager", "24"]]24
]51;["call", "Tapi_Vpager",  ["vpager", "25"]]25
]51;["call", "Tapi_Vpager",  ["vpager", "26"]]26
]51;["call", "Tapi_Vpager",  ["vpager", "27"]]27
]51;["call", "Tapi_Vpager",  ["vpager", "28"]]28
]51;["call", "Tapi_Vpager",  ["vpager", "29"]]29
]51;["call", "Tapi_Vpager",  ["vpager", "30"]]30
]51;["call", "Tapi_Vpager",  ["vpager", "31"]]31
]51;["call", "Tapi_Vpager",  ["vpager", "32"]]32
]51;["call", "Tapi_Vpager",  ["vpager", "33"]]33
]51;["call", "Tapi_Vpager",  ["vpager", "34"]]34
]51;["call", "Tapi_Vpager",  ["vpager", "35"]]35
]51;["call", "Tapi_Vpager",  ["vpager", "36"]]36
]51;["call", "Tapi_Vpager",  ["vpager", "37"]]37
]51;["call", "Tapi_Vpager",  ["vpager", "38"]]38
]51;["call", "Tapi_Vpager",  ["vpager", "39"]]39
]51;["call", "Tapi_Vpager",  ["vpager", "40"]]40
]51;["call", "Tapi_Vpager",  ["vpager", "41"]]41
]51;["call", "Tapi_Vpager",  ["vpager", "42"]]42
]51;["call", "Tapi_Vpager",  ["vpager", "43"]]43
]51;["call", "Tapi_Vpager",  ["vpager", "44"]]44
]51;["call", "Tapi_Vpager",  ["vpager", "45"]]45
]51;["call", "Tapi_Vpager",  ["vpager", "46"]]46
]51;["call", "Tapi_Vpager",  ["vpager", "47"]]47
]51;["call", "Tapi_Vpager",  ["vpager", "48"]]48
]51;["call", "Tapi_Vpager",  ["vpager", "49"]]49
]51;["call", "Tapi_Vpager",  ["vpager", "50"]]50
]51;["call", "Tapi_Vpager",  ["vpager", "51"]]51
]51;["call", "Tapi_Vpager",  ["vpager", "52"]]52
]51;["call", "Tapi_Vpager",  ["vpager", "53"]]53
]51;["call", "Tapi_Vpager",  ["vpager", "54"]]54
]51;["call", "Tapi_Vpager",  ["vpager", "55"]]55
]51;["call", "Tapi_Vpager",  ["vpager", "56"]]56
]51;["call", "Tapi_Vpager",  ["vpager", "57"]]57
]51;["call", "Tapi_Vpager",  ["vpager", "58"]]58
]51;["call", "Tapi_Vpager",  ["vpager", "59"]]59
]51;["call", "Tapi_Vpager",  ["vpager", "60"]]60
]51;["call", "Tapi_Vpager",  ["vpager", "61"]]61
]51;["call", "Tapi_Vpager",  ["vpager", "62"]]62
]51;["call", "Tapi_Vpager",  ["vpager", "63"]]63
]51;["call", "Tapi_Vpager",  ["vpager", "64"]]64
]51;["call", "Tapi_Vpager",  ["vpager", "65"]]65
]51;["call", "Tapi_Vpager",  ["vpager", "66"]]66
]51;["call", "Tapi_Vpager",  ["vpager", "67"]]67
]51;["call", "Tapi_Vpager",  ["vpager", "68"]]68
]51;["call", "Tapi_Vpager",  ["vpager", "69"]]69
]51;["call", "Tapi_Vpager",  ["vpager", "70"]]70
]51;["call", "Tapi_Vpager",  ["vpager", "71"]]71
]51;["call", "Tapi_Vpager",  ["vpager", "72"]]72
]51;["call", "Tapi_Vpager",  ["vpager", "73"]]73
]51;["call", "Tapi_Vpager",  ["vpager", "74"]]74
]51;["call", "Tapi_Vpager",  ["vpager", "75"]]75
]51;["call", "Tapi_Vpager",  ["vpager", "76"]]76
]51;["call", "Tapi_Vpager",  ["vpager", "77"]]77
]51;["call", "Tapi_Vpager",  ["vpager", "78"]]78
]51;["call", "Tapi_Vpager",  ["vpager", "79"]]79
]51;["call", "Tapi_Vpager",  ["vpager", "80"]]80
]51;["call", "Tapi_Vpager",  ["vpager", "81"]]81
]51;["call", "Tapi_Vpager",  ["vpager", "82"]]82
]51;["call", "Tapi_Vpager",  ["vpager", "83"]]83
]51;["call", "Tapi_Vpager",  ["vpager", "84"]]84
]51;["call", "Tapi_Vpager",  ["vpager", "85"]]85
]51;["call", "Tapi_Vpager",  ["vpager", "86"]]86
]51;["call", "Tapi_Vpager",  ["vpager", "87"]]87
]51;["call", "Tapi_Vpager",  ["vpager", "88"]]88
]51;["call", "Tapi_Vpager",  ["vpager", "89"]]89
]51;["call", "Tapi_Vpager",  ["vpager", "90"]]90
]51;["call", "Tapi_Vpager",  ["vpager", "91"]]91
]51;["call", "Tapi_Vpager",  ["vpager", "92"]]92
]51;["call", "Tapi_Vpager",  ["vpager", "93"]]93
]51;["call", "Tapi_Vpager",  ["vpager", "94"]]94
]51;["call", "Tapi_Vpager",  ["vpager", "95"]]95
]51;["call", "Tapi_Vpager",  ["vpager", "96"]]96
]51;["call", "Tapi_Vpager",  ["vpager", "97"]]97
]51;["call", "Tapi_Vpager",  ["vpager", "98"]]98
]51;["call", "Tapi_Vpager",  ["vpager", "99"]]99
]51;["call", "Tapi_Vpager",  ["vpager", "100"]]100
